tests/rag_answers/test_evaluate.py

```diff
@@ -12,6 +13,7 @@
 from govuk_chat_evaluation.rag_answers.evaluate import (
     AggregatedResults,
     evaluate_and_output_results,
+    _log_metric_errors,
```

Comment:
We shouldn't be testing this directly as we've indicated it's an internal method